tests/validation.bats

```diff
@@ -0,0 +1,112 @@
+#! /usr/bin/env bats
+
+# select test environment with
+# TARGET=tk bats validation.bats
+# or 
+# docker-compose exec -e TARGET=dc app bats -r tests/validation.bats
+case "${TARGET:-dc}" in
+  docker-compose | dc)
+    CMD="curl"
+    URLROOT="http://localhost"
+    ;;
+  production)
+    CMD="curl"
+    URLROOT="https://labs.data.gov/dashboard"
+    ;;
+  test-kitchen | tk | * )
```

Comment:
FYI, test-kitchen is going away soon, to be replaced with molecule. I think this was useful for your debugging, but do you think it would be worth re-implementing for molecule?

Reply:
Probably not. I needed this while I was still working out templating issues between docker-compose and ansible. So it may be of little value in molecule, and even less value when on cloud.gov.